LibreOfficeWriter_Doc.au3

```diff
@@ -8,23 +8,12 @@
 #include "LibreOfficeWriter_Par.au3"
 
 ; #INDEX# =======================================================================================================================
-; Title .........: Libre Office Writer (LOWriter)
+; Title .........: LibreOffice UDF
 ; AutoIt Version : v3.3.16.1
-; UDF Version    : 0.0.0.3
-; Description ...: Provides basic functionality through Autoit for interacting with Libre Office Writer.
+; Description ...: Provides basic functionality through AutoIt for Creating, Modifying, Closing, Saving, Searching, etc. L.O. Writer documents.
```

Comment:
documents ?

Reply:
You would prefer singular? Or what are your thoughts? -- I'm trying to define what each of your single files are for. Bad idea?

Reply:
Here was my mistake.
Sorry.